frontend/src/components/regions/AddCardReviewModal/Presentation.js

```diff
@@ -38,12 +38,32 @@ const useStyles = makeStyles((theme) => ({
   rightButton: {
     float: "right",
   },
+  popUpInfo: {
+    display:"block",
+  },
+  
+  helpPopUpInfo: {
+    textAlign: "right",
+    marginTop: "8px",
+    marginRight: "12px",
+    fontSize: "18px",
+    "&:hover": {
+      color: "#A8A8A8",
+    },
+  },
 }));
 
-const StatusHelp = () => {
+const StatusHelp = ({ closePopUpInfo, closeHelpPopUp }) => {
+const classes = useStyles();
+  
   return (
     <Help buttonText="How do I choose a status?">
-      <Paper>
+
+      <Paper ref={closePopUpInfo} className={classes.popUpInfo}>
+        <Typography variant="subtitle2" className={classes.helpPopUpInfo} onClick={closeHelpPopUp}>
+          X
```

Comment:
Hi there @Vuyisanani , not make use of the exit icon found in the material UI icons ?